bindings/ruby/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# this is set in the parent directory, but for some reason it does not get inherited
```

Comment:
All this CMakeLists.txt are missing the licence. @drdanz do you think this could be a problem?

Reply:
Yes, please add the original license in the old CMakeLists.txt to all these files...